SPECS/python-ssh-python/python-ssh-python.spec

```diff
@@ -0,0 +1,102 @@
+%{?python_enable_dependency_generator}
+%global modname ssh-python
+Name:           python-%{modname}
+Version:        0.9.0
+Release:        3%{?dist}
+Summary:        Bindings for libssh C library
+License:        LGPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/ParallelSSH/ssh-python
+Source0:        %{url}/archive/refs/tags/%{version}.tar.gz#/%{modname}-%{version}.tar.gz
+
+Patch0001:      0001-Set-master_doc-to-index-in-conf.py-for-sphinx.patch
+
+BuildRequires:  cmake
+BuildRequires:  gcc
+BuildRequires:  libssh-devel
+BuildRequires:  python3-devel
+BuildRequires:  python3-setuptools
+BuildRequires:  python3-Cython
+BuildRequires:  python3-sphinx
+BuildRequires:  python3-sphinx_rtd_theme
+
+%if %{with_check}
+BuildRequires:  python3-pytest
+BuildRequires:  %{_sbindir}/sshd %{_bindir}/ssh-agent
```

Comment:
Please put these build requirements on separate lines for readability.

Reply:
I'd suggest running the linter for the spec files, which should do it for you. It **may** make some changes, which we don't want (surround macros depending on parameters or replace source-specific paths with Mariner SRPM macros), so I'd double-check the results of running the linter, but most of the modifications will be helpful.

Reply:
Done.